app/src/main/java/asd/org/ahorcado/activities/InputActivity.java

```diff
@@ -34,13 +52,70 @@ public void guessLetter(View view) {
         char letter = button.getText().toString().toCharArray()[0];
         TextView textViewWord = (TextView) findViewById(R.id.word);
         textViewWord.setText(match.execute(letter));
+        AlertDialog.Builder dialog = new AlertDialog.Builder(this);
+        dialog.setIcon(android.R.drawable.ic_dialog_info);
         try {
             if (match.afterExecute(letter)) {
-                //Win
+                createDialog(dialog, R.string.win_game, this);
             }
         } catch (MatchLostException e) {
-            //Lost
+            createDialog(dialog, R.string.lost_game, this);
+        } finally {
+            ProgressBar progressBar = (ProgressBar) findViewById(R.id.progressBar);
+            progressBar.setProgress(match.getLife());
+            setBackground();
+        }
+    }
+
+    private void setInitialActivity(){
+        setContentView(R.layout.activity_input);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        TextView textViewWord = (TextView) findViewById(R.id.word);
+        textViewWord.setText(match.getNewWord());
+        ProgressBar progressBar = (ProgressBar) findViewById(R.id.progressBar);
+        progressBar.setMax(match.getLife());
+        progressBar.setProgress(match.getLife());
+    }
+
+    private void createDialog(AlertDialog.Builder dialog, int title, final InputActivity thisActivity) {
+        dialog.setTitle(title)
+                .setPositiveButton(R.string.play_button, new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int which) {
+                        finish();
+                        startActivity(new Intent(thisActivity, InputActivity.class));
+                    }
+                })
+                .setNegativeButton(R.string.cancel_button, new DialogInterface.OnClickListener() {
+                    public void onClick(DialogInterface dialog, int which) {
+                        finish();
+                        startActivity(new Intent(thisActivity, MainActivity.class));
+                    }
+                }).show();
+    }
+
+    private void setBackground() {
+        RelativeLayout relativeLayout = (RelativeLayout) findViewById(R.id.input_layout);
```

Comment:
What do you think if you put this code in other class? Because in my opinion the Activity class should not has the responsability to decide how to print or put differents "views games". I'm agree to call for from this class, but in my opinion these behaviours could be in another class. 
Also you could implement Strategy pattern on this one, but it is up to you to decide.